app/helpers/filters_helper.rb

```diff
@@ -184,4 +237,48 @@ def filters_count(filters)
   def year_combo(year)
     "#{year}/#{year - 2000 + 1}"
   end
+
+  def formatted_years_filter(session_filters)
+    return unanswered_filter_value if session_filters["years"].blank?
+
+    session_filters["years"].map { |year| year_combo(year.to_i) }.to_sentence
+  end
+
+  def formatted_status_filter(session_filters)
+    return unanswered_filter_value if session_filters["status"].blank?
+
+    session_filters["status"].map { |status| status_filters[status] }.to_sentence
+  end
+
+  def formatted_needstype_filter(session_filters)
+    return unanswered_filter_value if session_filters["needstypes"].blank?
+
+    session_filters["needstypes"].map { |needstype| needstype_filters[needstype] }.to_sentence
+  end
+
+  def formatted_assigned_to_filter(session_filters)
+    return unanswered_filter_value if session_filters["assigned_to"].blank?
+    return "All" if session_filters["assigned_to"].include?("all")
+    return "You" if session_filters["assigned_to"].include?("you")
+
+    user = User.find(session_filters["user"].first)
+    "#{user.name} (#{user.email})"
+  end
+
+  def formatted_owned_by_filter(session_filters)
+    return "All" if params["id"].blank? && (session_filters["owning_organisation"].blank? || session_filters["owning_organisation"]&.include?("all"))
+
+    session_org_id = session_filters["owning_organisation"].is_a?(Array) ? session_filters["owning_organisation"].first : session_filters["owning_organisation"]
```

Comment:
What makes this sometimes an array? (I don't think I've touched filters much)

Reply:
You ask good questions, I can't reproduce it and the tests don't seem to fail either. I'll remove it for now, it's likely I've added it cause I ran into a case that I ended up removing